docs/documentation/maya/skin.md

```diff
@@ -1,127 +1,185 @@
+- **Hard Constraints**: Weight to modulate the correction applied to the vertices to keep them at a constant transformation, local to the closest point on the reference mesh at initialization. Hard constraint maps will force the geometry points to keep the original position. A low value of hard constraints may be desired to allow the skin to create wrinkles, sliding, etc.
 
-<figure>
-  <img src="images/skin_paint_example.png" caption="AdonisFX Paint Tool"> 
-  <figcaption>Figure 1: Example of painted weights on the skin of a bear character. From left to right: Hard Constraints, Slide Constraints and Soft Constraints.</figcaption>
-</figure>
+    - *Tip*: Flood the geometry with a very low value 0.1 - 0.2. Give a value of 1.0 to the edges of the skin to guarantee that’s perfectly attached to the target geometry
 
-> [!NOTE]
-> - *Hard*, *Soft* and *Slide* values are normalized for each vertex. Make sure to paint the values that you want to give priority to at the end in order to avoid the internal normalization override them in further strokes.
+    - *Tip*: Smooth the borders by using the smooth-Flood combination to make sure the simulation doesn’t find hard edges as this could create odd wrinkles (unless that’s something you are looking for!).
 
-### AdonisFX Paint Tool
+- **Soft Constraints**: Weight to modulate the correction applied to the vertices to keep them at a constant distance to the closest point on the reference mesh at initialization. Painting these constraint weights would allow the deformer to create a wrinkle effect when combined with hard and slide weights.
 
-To configure the paintable attributes in the skin deformer, the AdonisFX paint tool must be used. Apart from the standard functionalities that the Maya default paint context provides, this tool also processes the painted weights to guarantee that the requirements of solver are satisfied.
+    - *Tip*: Flood the geometry with a very low value 0.1 - 0.2.
 
-<figure>
-  <img src="images/paint_tool.png" caption="AdonisFX Paint Tool"> 
-  <figcaption>Figure 2: AdonisFX Paint Tool</figcaption>
-</figure>
+    - *Tip*: It may be desirable to leave this map weight to 1 as a starting point and tweak some areas later on as the results of the skin simulation are seen.
+
+- **Slide Constraints**: Weight to modulate the correction applied to the vertices to keep them at a constant distance to the reference mesh sliding along the reference surface. In the example of a biped or quadruped, it is recommended to set a value of 1.0 on the scapulas, shoulders, elbows and knees and an overall value of 0 on the rest of the body.
+
+    - *Tip*: Smooth the borders by using the Smooth-Flood combination to make sure the simulation doesn’t find hard edges as this could create odd looking wrinkles (unless that’s something you are looking for!).
```

Comment:
```suggestion
    - *Tip*: Smooth the borders by using the Smooth and Flood combination to make sure the simulation does not find hard edges as this could create odd looking wrinkles (unless that is something you are looking for!).
```